lib/bloc/csv_exporter.dart

```diff
@@ -114,9 +114,9 @@ class CsvExporter {
 
   String _appendFilterInformation(String filePath) {
     _log.info("add filter information to path started");
-    if (filter == PaymentTypeFilter.Sent) {
+    if (!filter.contains(PaymentTypeFilter.Received)) {
       filePath += "_sent";
-    } else if (filter == PaymentTypeFilter.Received) {
+    } else if (!filter.contains(PaymentTypeFilter.Sent)) {
       filePath += "_received";
     }
```

Comment:
Think this logic should be changed to sth like:

```
    List<PaymentTypeFilter>? filterList = filters;
    if (filterList != null && filterList != PaymentTypeFilter.values) {
      for (var filter in filterList) {
        switch (filter) {
          case PaymentTypeFilter.Sent:
            filePath += "_sent";
            break;
          case PaymentTypeFilter.Received:
            filePath += "_received";
            break;
          case PaymentTypeFilter.ClosedChannels:
            filePath += "_closed_channels";
            break;
        }
      }
    }
```

Where filter information is appended unless `filters` is empty or null(_unfiltered_), or filters has all `PaymentTypeFilter.values` (_unfiltered_)

Reply:
Thanks, I've added this filtering, but added closed channels to the sent case.